src/services/MaoriTranslationService.php

```diff
@@ -32,9 +32,10 @@ public function enabled(): bool
      * related to the keywords
      *
      * @param string $wordOrSentence
+     * @param AudioDefinition|null $object
      * @return array
      */
-    public function getDefinitionAndAudio($wordOrSentence): array
+    public function getDefinitionAndAudio($wordOrSentence, $object = null): array
```

Comment:
Sorry, I think it would be cleaner to just pass the object.
And in the MaoriTranslationService, change $wordOrSentence to $object->Term. Also cast the $object to be an AudioDefinition.